src/profile-logic/profile-data.js

```diff
@@ -833,6 +836,8 @@ export function invertCallstack(thread: Thread): Thread {
 
     const oldStackToNewStack = new Map();
 
+    // For one specific stack this will loop over its prefixes and create new
```

Comment:
maybe "this will ensure that stacks are created for all of its ancestors, by walking its prefix chain up to the root"?